src/selectors.ts

```diff
@@ -1,11 +1,11 @@
 import { SelectableBaseState, AjaxState, BaseState } from './types'
 
 function selectedEntity<T>(state: SelectableBaseState<T>): T | null {
-  if (!state.selectedId) {
-    return null
-  }
+  return selectEntity(state, state.selectedId)
+}
 
-  return state.all[state.selectedId]
+function selectEntity<T>(state: BaseState<T>, id: any): T | null {
```

Comment:
hm, should this be `id: string | number`?

Reply:
Yes! Done.